### DISCUSSION TITLE
Choosing algorithm

### DISCUSSION THREAD

**Discussion by eishwezin20** [tensorflow/decision-forests#191] [Q&A] [answered]
TensorFlow Decision Forests offer different algorithms like Random Forest and Gradient Boosted Decision Trees. How do I choose the right algorithm for my problem?

**Comment by rstz** [comment, marked as answer]
There are many high-quality articles and blog posts on this topic, see [here](https://stats.stackexchange.com/questions/173390/gradient-boosting-tree-vs-random-forest) for a discussion and more references.

_Note: Our team deleted a (partially) off-topic answer to this question_
